pysys/basetest.py

```diff
@@ -327,7 +322,7 @@ def wait(self, interval):
 
 
 	# test validation methods.
-	def assertThat(self, conditionstring, *args):
+	def assertThat(self, conditionstring, *args, **kwargs):
```

Comment:
Why is `assertThat` different and still using kwargs when the variable argument lists have been removed from all the other `assert*` methods?

Reply:
Purely because I was forced to - in Python 2 keyword arguments can't be used togetehr with *args varargs as it gives a syntax error (it's fine in python 3) - cf https://docs.python.org/2/reference/compound_stmts.html#function-definitions

The other cases don't have *args so can be done the normal simple way, but because this specific function uses *args we need use **kwargs for python 2's sake. But at least the way you invoke it as a user is still simple. :)

Reply:
Thanks, that's something I have learned today.